docs/UserGuide.md

```diff
@@ -150,7 +150,13 @@ Format: `exit`
 
 ### Saving the data
 
-Hall-y's data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+
+### Exporting of email : `export`
+
+Exports the email address of all entries in Hall-y as a .txt file.
+
+Format: `export`
```

Comment:
Hmm seems like there was an issue with Git. Could you keep the original line under 'Saving the Data'?

Also, could you update the command summary at the bottom of the UG as well?

Reply:
Yep will do thanks